requirements.txt

```diff
@@ -1,2 +1,3 @@
 Scrapy==2.11.1
 zyte-spider-templates==0.7.1
+scrapy-zyte-api[provider]>=0.17.0
```

Comment:
```suggestion
scrapy-zyte-api[provider]>=0.17.0
zyte-spider-templates==0.7.1
```

Reply:
Actually, I wanted to ask about this whether we want to `==0.17.0` here, `>=0.17.0` here or change `>=0.16.0` to `>=0.17.0` in zuyte-spider-templates?

Reply:
In zyte-spider-templates it may be best to keep it frozen, as zyte-spider-templates may implement changes not compatible with Scrapy Cloud.

As for the rest, I am slightly in favor of not freezing, so that we do not have to update this file after every release of a package, but no strong opinion.